plugins/defaultplugins/l3plugin/model/l3/l3.proto

```diff
@@ -3,17 +3,17 @@ syntax = "proto3";
 package l3;
 
 message StaticRoutes {
-    message Ip {
+    message Route {
         uint32 vrf_id = 1;                   /* VRF identifier */
         string description = 2;              /* optional description */
-        string destination_address = 3;   /* ip address + prefix in format <address>/<prefix> */
-
-        message NextHop {
+        string destination_address = 3;      /* ip address + prefix in format <address>/<prefix> */
+        bool multipath = 4;                  /* enable/disable multipath */
```

Comment:
Would it be possible to ommit this field? If there are multiple hops it is automatically set to true, false otherwise.